src/plugins/workspace/public/components/use_case_overview/setup_overview.scss

```diff
@@ -0,0 +1,3 @@
+.analyticsGettingStartedWorkspaceCardsIcon {
+  color: $euiColorMediumShade;
```

Comment:
Should it just use the `color` props of `<OuiIcon />`?

Reply:
<img width="303" alt="image" src="https://github.com/user-attachments/assets/3246fb62-8b8a-450d-8511-b8aea027064d">
Actually I tried all of the possible values here but none of the color meet our expectation so I had to create scss file for each icon.